scripts/ubuntu-bartender/gce-provider

```diff
@@ -0,0 +1,79 @@
+function _wait-instance-connectable() {
```

Comment:
this is now the 3rd copy of that function. could you move that to something like `common-provider` where we can collect common functions?
Not a blocker but would be nice to have

Reply:
It is slightly different from the rest of them. I'll do some refactoring in a separate PR.